forge-gui/res/cardsfolder/upcoming/vren_the_relentless.txt

```diff
@@ -0,0 +1,10 @@
+Name:Vren, the Relentless
+ManaCost:2 U B
+Types:Legendary Creature Rat Rogue
+PT:3/4
+K:Ward:2
+R:Event$ Moved | ActiveZones$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.OppCtrl | ReplaceWith$ Exile | Description$ If a creature an opponent controls would die, exile it instead.

```

Comment:
Crash ⚠️

Reply:
I forgot Origin$, could that be it?

Reply:
non-existing subability...

Reply:
But there is no subability in that code, where is it required?

Reply:
what's ReplaceWith...?

Reply:
Sorry, Rest in Peace used just "Exile" for the ability, so I haven't noticed it's a subability and not just a zone.